src/Typeahead.jsx

```diff
@@ -410,7 +410,7 @@ Typeahead.propTypes = {
 Typeahead.defaultProps = {
   isLoading: false,
   editable: true,
-  empty: '',
+  empty: false,
```

Comment:
I see this code in OneSony:
`      <Typeahead
        clearOnSelect
        placeholder='Search'
        empty='No projects found...'`
Thoughts @isuttell ?

Reply:
Actually, I guess that's the original non-ship-components code. False alarm.